js/data/scripts/createTweetsOl.js

```diff
@@ -0,0 +1,12 @@
+"use strict";
+
+function createTweetOl(tweet) {
```

Comment:
This function creates an `<ol>` which only contains a single tweet.
It should take an array of tweets as argument and return an `<ol>` with one `<li>` **per** tweet